### ISSUE TITLE
display SPARQL source in caption

### ISSUE THREAD

**Issue by smurp** [cwrc/HuViz#319]
### Context

When SPARQL LOAD has happened the caption under the graph is not displaying the source.

It is not quite clear what to display there though!

The top of the **Commands** tab is showing 

* Endpoint
* Graph
* Item
* Limit

What is the spirit of the caption?  To show links? To show labels which remain in the picture?

### Proposal

Let's display links to `endpoint`, `graph` and `item`.  

Note that: 

* `item` is not available if one of the Graphs was picked and then LOADed.  
* `graph` might be the same as `endpoint` if the none of the graphs are chosen
* `graph` might be missing (maybe?) if no graphs were reported by the search for graphs

So, in other words, of `endpoint`, `graph` and `item` we should probably only show the ones which populated and unique.

Thoughts?

@SusanBrown @antimony27 @wolfmaul

**Comment by smurp** [comment]
Thinking about all this it occurs to me that we should make the legend up at the top of the **Commands** tab be not just labels but links to the resources.
